.github/workflows/rails-ci.yml

```
@@ -52,6 +52,6 @@ jobs:
         run: |
           bundle exec rails db:create db:schema:load
       - name: Run RuboCop
-        run: bundle exec rubocop
+        run: bundle exec rubocop 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)